exist-core/src/main/java/org/exist/launcher/WindowsServiceManager.java

```diff
@@ -109,7 +109,7 @@ public void install() throws ServiceManagerException {
                 "--LogPrefix=service",
                 "--PidFile=service.pid",
                 "--Startup=auto",
-                "--Jvm=auto",
+                "--Jvm=" + findJvm().orElse("auto"),
```

Comment:
Do you plan to submit an issue upstream to procrun too?

Reply:
There already is a feature request and some discussion around the topic, which suggests the solution used in this PR. The documented behaviour of `--Jvm=auto` though is to use the registry to determine the location of `jvm.dll`, which for the mentioned Java distributions does not work correctly. It is disputable who's right or wrong here, but since we can easily determine the correct location, we should do so.

Reply:
Okay cool. Totally agree